### ISSUE TITLE
Xpath with hierarchical relationships is not recognized 

### ISSUE THREAD

**Issue by chenqinggang001** [seleniumbase/SeleniumBase#1526]
```log
Testing started at 13:44 ...
Launching pytest with arguments test_04_marking_stu_exam.py::TestMarkingStuExam::test_makring_stu_exam --no-header --no-summary -q in D:\projects\utestweb-autotest\testcase\tea_testcase\exam

============================= test session starts =============================
collected 1 item                                                               

test_04_marking_stu_exam.py::TestMarkingStuExam::test_makring_stu_exam FAILED [100%]//span[@title='测试考试2022-09-26']/../../..//button[@class='btn btn-default dropdown-toggle']
2022-09-26 13:44:29.040 | INFO     | common.utility:hook_logs:72 - {'session_id': 'f5e0b571247395b691949698b96d9a90', 'test_id': 'test_04_marking_stu_exam.TestMarkingStuExam.test_makring_stu_exam', 'version': '(4, 3, 1)', 'bower_version': '105.0.5195.102'}

testcase\tea_testcase\exam\test_04_marking_stu_exam.py:22 (TestMarkingStuExam.test_makring_stu_exam)
self = <test_04_marking_stu_exam.TestMarkingStuExam testMethod=test_makring_stu_exam>

    def test_makring_stu_exam(self):
>       self.init_tea_page()

test_04_marking_stu_exam.py:24: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
test_04_marking_stu_exam.py:18: in init_tea_page
    self.hover_on_element(exam_el, by="xpath")
C:\Users\chenqinggang\.virtualenvs\utestweb-autotest-DixkMSpk\lib\site-packages\seleniumbase\fixtures\base_case.py:2132: in hover_on_element
    selector = self.convert_to_css_selector(selector, By.XPATH)
C:\Users\chenqinggang\.virtualenvs\utestweb-autotest-DixkMSpk\lib\site-packages\seleniumbase\fixtures\base_case.py:6593: in convert_to_css_selector
    return self.convert_xpath_to_css(selector)
C:\Users\chenqinggang\.virtualenvs\utestweb-autotest-DixkMSpk\lib\site-packages\seleniumbase\fixtures\base_case.py:6575: in convert_xpath_to_css
    return xpath_to_css.convert_xpath_to_css(xpath)
C:\Users\chenqinggang\.virtualenvs\utestweb-autotest-DixkMSpk\lib\site-packages\seleniumbase\fixtures\xpath_to_css.py:226: in convert_xpath_to_css
    _get_raw_css_from_xpath(xpath_section, original)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

xpath = "//span[@title='测试考试2022-09-26']/../../.."
original = "//span[@title='测试考试2022-09-26']/../../..//button[@class='btn btn-default dropdown-toggle']"

    def _get_raw_css_from_xpath(xpath, original):
        css = ""
        attr = ""
        position = 0
    
        while position < len(xpath):
            node = prog.match(xpath[position:])
            if node is None:
>               raise XpathException(
                    "\nInvalid or unsupported XPath:\n%s\n"
                    "(Unable to convert XPath Selector to CSS Selector)"
                    "" % original
                )
E               seleniumbase.fixtures.xpath_to_css.XpathException: 
E               Invalid or unsupported XPath:
E               //span[@title='测试考试2022-09-26']/../../..//button[@class='btn btn-default dropdown-toggle']
E               (Unable to convert XPath Selector to CSS Selector)

C:\Users\chenqinggang\.virtualenvs\utestweb-autotest-DixkMSpk\lib\site-packages\seleniumbase\fixtures\xpath_to_css.py:91: XpathException
```

**Comment by chenqinggang001** [comment]
This xpath is valid on the page：
![image](https://user-images.githubusercontent.com/74290987/192203578-fb36d064-7c15-4f2b-8587-7a265fe67dcc.png)

**Comment by chenqinggang001** [comment]
It's fine for me to use `click`, maby ​have something to do with the `hover_on_elemen`

**Comment by mdmintz** [comment]
@chenqinggang001 
That issue was fixed 19 days ago in version ``4.3.6`` - https://github.com/seleniumbase/SeleniumBase/commit/aa0e3243f3ae814dd9977bf5b4a15ccf1016a687
(https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.3.6)
Please upgrade to a newer version of SeleniumBase.
